x/staking/keeper/validator.go

```diff
@@ -385,7 +381,7 @@ func (k Keeper) UnbondAllMatureValidatorQueue(ctx sdk.Context) {
 		for _, valAddr := range timeslice {
 			val, found := k.GetValidator(ctx, valAddr)
 			if !found {
-				continue
+				panic("validator in the unbonding queue was not found")
```

Comment:
As explained in person, not sure if this change makes sense

Reply:
`RemoveValidator` now removes the validator from the unbonding queue - thoughts?

Reply:
didn't we require this for determining unbonding-delegations or redelegations for slashing purposes? If that is not the case then this comment is resolved

Reply:
require what? the validator would still be around in that case, right?